components/script/dom/htmlmeterelement.rs

```diff
@@ -79,18 +75,8 @@ impl HTMLMeterElement {
 
     fn create_shadow_tree(&self, can_gc: CanGc) {
         let document = self.owner_document();
-        let root = self
-            .upcast::<Element>()
-            .attach_shadow(
-                IsUserAgentWidget::Yes,
-                ShadowRootMode::Closed,
-                false,
-                false,
-                false,
-                SlotAssignmentMode::Manual,
-                can_gc,
-            )
-            .expect("Attaching UA shadow root failed");
+        // FIXME: Add servo specific appearence reftest
```

Comment:
Adding a reftest isn't something that affects this code. maybe file an issue instead of the comment.

Reply:
I will file an issue later for all of these. It is outside of the scope for this PR, but it would be nice to document these kind of things.

Reply:
The best place to document tasks for the future is to file an issue. It might be outside the scope of the PR, but that doesn't mean it cannot be done in parallel with a PR that is in process. The main point here is that adding a code comment isn't an appropriate place for this kind of reminder.